velox/functions/prestosql/ArrayFunctions.h

```diff
@@ -418,4 +418,51 @@ struct ArrayFrequencyFunction {
   }
 };
 
+template <typename TExecParams, typename T>
+struct ArrayNormalizeFunction {
+  VELOX_DEFINE_FUNCTION_TYPES(TExecParams);
+
+  FOLLY_ALWAYS_INLINE void callNullFree(
+      out_type<velox::Array<T>>& result,
+      const null_free_arg_type<velox::Array<T>>& inputArray,
+      const null_free_arg_type<T> p) {
```

Comment:
@isadikov Got a lint error. Would you take a look?

<img width="514" alt="Screenshot 2023-02-09 at 8 49 37 AM" src="https://user-images.githubusercontent.com/27965151/217830899-c4611713-56ac-4671-8870-19cbb5c69d59.png">

Reply:
I cannot share a link as it is not accessible outside of Meta. To fix, just pass p as const &:

```
const null_free_arg_type<T>& p
```

Reply:
Yes, let me fix it.